### DISCUSSION TITLE
How to apply Serverless on AWS Lambda?

### DISCUSSION THREAD

**Discussion by XOneto9** [vercel/next.js#18181] [Help]
Hi, I have some question about serverless on Lambda

Now, in my case, I deployed AWS ElasticBeanstalk because of fail setup Lambda.

The problem is I think lots of developers want to adopt lambda for serverless, But after update from nextjs 8 to nextjs 9, The Next Module is supported to serverless on AWS lambda@edge with CloudFront for super-fast loading anywhere in the world. 

It's cool!, however, The lambda@edge service only works on just us-ease-1(Virginia).
And Lambda need to VPC and NetGatway if want to connect RDS (it can not connect another region network)

also, Many applications or websites need to Databases like AWS rds or Aurora. but original source file or source place on the Virginia region AWS. And source file copy replica version on the world CloudFront region.

The problem is in the case Database can not connect with service becuase the real source file is placed in Virginia region AWS. 
So, In Other places like Europe, Aisa or other countries can not access RDS.

That's it 

I need to your opinion and ideas! Thanks 

****************
Now using
1. nextjs9
2. RDS mysql
3. region Aisa/Seoul Region

Wanted
1. to apply lambda with RDS

**Comment by sladg** [comment]
Here you go :) 
https://github.com/sladg/nextjs-lambda

tested with VPC's and RDS in private subnet, no problem.
You will, however, need to upgrade to newer Next
